### DISCUSSION TITLE
Error when uploading new document

### DISCUSSION THREAD

**Discussion by jediknight36** [pashpashpash/vault-ai#67] [Q&A]
I am trying to upload a document, but keep getting this error, Im guessing says my database is full:
Failed Files:
2022 Pierce County Local Rules.pdf
Reason: Error upserting embeddings to Pinecone: {"code":8,"message":"index is full, cannot accept data.","details":[]}

When I try, the document count goes up, but it still gives me the same error. This is where it stands now:

Your Usage this Month
9/200 Questions
11.91 MB/1600 MB Uploaded
1201/10000 Pages

The pdf Im trying to upload is less than 1mb. 

What is causing this and how do I work around it?

Thank you in advance.

**Comment by Steake** [comment]
same issue here `Failed Files:

    filelist.go
    Reason: Error upserting embeddings to Pinecone: {"code":8,"message":"index is full, cannot accept data.","details":[]}`

**Comment by pashpashpash** [comment]
@jediknight36 @Steake This issue should be fixed now

**Reply by jediknight36** [reply]
Thank you!

**Comment by Aben25** [comment]
Sorry can we reschedule it for next week?
Still on a meeting
